aea/connections/stub/readme.md

```diff
@@ -0,0 +1,7 @@
+# stub connection
+A simple connection for communication with an AEA, using the file system as a point of data exchange.
+
+## Usage
+First, add the connection to your AEA project: `aea add connection fetchai/stub:0.6.0`. (If you have created your AEA project with `aea create` then the connection will already be available by default.)
```

Comment:
@DavidMinarsch this must be updated whenever the version number of `fetchai/stub` bumps up. Does the script also read from README files?

Reply:
yes it does